script/fork-helpers/PauseResume.s.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-FileCopyrightText: 2024 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity 0.8.24;
+
+import "forge-std/Script.sol";
+import { DeploymentFixtures } from "test/helpers/Fixtures.sol";
+import { ForkHelpersCommon } from "./Common.sol";
+
+contract PauseResume is Script, DeploymentFixtures, ForkHelpersCommon {
+    address internal csmAdmin;
+    address internal accountingAdmin;
+
+    modifier broadcastCSMAdmin() {
+        _setUp();
+        csmAdmin = _prepareAdmin(address(csm));
+        vm.startBroadcast(csmAdmin);
+        _;
+        vm.stopBroadcast();
+    }
+
+    modifier broadcastAccountingAdmin() {
+        _setUp();
+        accountingAdmin = _prepareAdmin(address(accounting));
+        vm.startBroadcast(accountingAdmin);
+        _;
+        vm.stopBroadcast();
+    }
+
+    function pauseCSM() external broadcastCSMAdmin {
+        csm.grantRole(csm.PAUSE_ROLE(), csmAdmin);
+        csm.pauseFor(type(uint256).max);
+
+        assertTrue(csm.isPaused());
```

Comment:
if we have asserts here, do we need the same in another script (`NodeOperators.s.sol`)?

Reply:
yep!